lib/elements.hpp

```diff
@@ -63,3677 +61,4148 @@ int getElementMainIsotope(int Z);
-{ 277, { 277.153483, NAN, NAN } },
-{ 278, { 278.156454, NAN, NAN } },
-{ 279, { 279.158343, NAN, NAN } },
-} } },
-{ "Ds", { 110, 281, {
-{ 267, { 267.143726, 1.5, NAN } },
-{ 268, { 268.143477, 0.0, NAN } },
-{ 269, { 269.144751021, 4.5, NAN } },
-{ 270, { 270.14458309, 0.0, NAN } },
-{ 271, { 271.145946, 4.5, NAN } },
-{ 272, { 272.146018, 0.0, NAN } },
-{ 273, { 273.148531, 1.5, NAN } },
-{ 274, { 274.149434, 0.0, NAN } },
-{ 275, { 275.151976, NAN, NAN } },
-{ 276, { 276.153024, 0.0, NAN } },
-{ 277, { 277.155815, 0.5, NAN } },
-{ 278, { 278.157146, 0.0, NAN } },
-{ 279, { 279.160093, NAN, NAN } },
-{ 280, { 280.16159, 0.0, NAN } },
-{ 281, { 281.164715, 1.5, NAN } },
-} } },
-{ "Rg", { 111, 283, {
-{ 272, { 272.153273, 5.0, NAN } },
-{ 273, { 273.153189, NAN, NAN } },
-{ 274, { 274.155249, NAN, NAN } },
-{ 275, { 275.155981, NAN, NAN } },
-{ 276, { 276.158333, NAN, NAN } },
-{ 277, { 277.159247, NAN, NAN } },
-{ 278, { 278.161587, NAN, NAN } },
-{ 279, { 279.162937, NAN, NAN } },
-{ 280, { 280.165203, NAN, NAN } },
-{ 281, { 281.166718, NAN, NAN } },
-{ 282, { 282.169405, NAN, NAN } },
-{ 283, { 283.170995, NAN, NAN } },
-} } },
-{ "Cn", { 112, 285, {
-{ 276, { 276.16141, 0.0, NAN } },
-{ 277, { 277.163611, 1.5, NAN } },
-{ 278, { 278.164179, 0.0, NAN } },
-{ 279, { 279.166432, NAN, NAN } },
-{ 280, { 280.167147, 0.0, NAN } },
-{ 281, { 281.169641, 1.5, NAN } },
-{ 282, { 282.170668, 0.0, NAN } },
-{ 283, { 283.173362, NAN, NAN } },
-{ 284, { 284.174499, 0.0, NAN } },
-{ 285, { 285.177321, 2.5, NAN } },
-} } },
-{ "Ed", { 113, 287, {
-{ 278, { 278.170574, NAN, NAN } },
-{ 279, { 279.17095, NAN, NAN } },
-{ 280, { 280.172991, NAN, NAN } },
-{ 281, { 281.17371, NAN, NAN } },
-{ 282, { 282.175766, NAN, NAN } },
-{ 283, { 283.17682, NAN, NAN } },
-{ 284, { 284.178843, NAN, NAN } },
-{ 285, { 285.180066, NAN, NAN } },
-{ 286, { 286.182518, NAN, NAN } },
-{ 287, { 287.18384, NAN, NAN } },
-} } },
-{ "Fl", { 114, 289, {
-{ 284, { 284.181344, 0.0, NAN } },
-{ 285, { 285.183579, NAN, NAN } },
-{ 286, { 286.184406, 0.0, NAN } },
-{ 287, { 287.186875, NAN, NAN } },
-{ 288, { 288.187916, 0.0, NAN } },
-{ 289, { 289.190623, 2.5, NAN } },
-} } },
-{ "Ef", { 115, 291, {
-{ 287, { 287.190978, NAN, NAN } },
-{ 288, { 288.192992, NAN, NAN } },
-{ 289, { 289.193953, NAN, NAN } },
-{ 290, { 290.196345, NAN, NAN } },
-{ 291, { 291.197522, NAN, NAN } },
-} } },
-{ "Lv", { 116, 293, {
-{ 289, { 289.198099, 2.5, NAN } },
-{ 290, { 290.198818, 0.0, NAN } },
-{ 291, { 291.201169, NAN, NAN } },
-{ 292, { 292.202086, 0.0, NAN } },
-{ 293, { 293.204691, NAN, NAN } },
-} } },
-{ "Eh", { 117, 294, {
-{ 291, { 291.205906, NAN, NAN } },
-{ 292, { 292.207812, NAN, NAN } },
-{ 293, { 293.20868, NAN, NAN } },
-{ 294, { 294.210974, NAN, NAN } },
-} } },
-{ "Ei", { 118, 295, {
-{ 293, { 293.213498, 0.5, NAN } },
-{ 294, { 294.214132, 0.0, NAN } },
-{ 295, { 295.216332, NAN, NAN } },
-} } },
+const map<string, element> atomic_data = {{ "H", {
+      1, 1, {
```

Comment:
Tried to leave a comment here as part of review but it didn't go through:
Like econfig, could make the first entry here more similar to the others by bumping it to a newline:
```suggestion
const map<string, element> atomic_data = {
  {
    "H", {
      1, 1, {
```